index.html

```diff
@@ -255,6 +255,62 @@
 
 ---
 
+# Continuous Analysis
+
+The Status Quo Doesn't Work:
+.task[
```

Comment:
Ahh - this presentation actually is the pre-meeting component. In person is a case study. It's the last .task[] item.